src/Xamarin.Android.Build.Tasks/Tasks/JavaDependencyVerification.cs

```diff
@@ -331,18 +332,26 @@ public MicrosoftNuGetPackageFinder (string? file, TaskLoggingHelper log)
 
 	public class PackageListFile
 	{
-		[JsonProperty ("packages")]
+		[JsonPropertyName ("packages")]
 		public List<Package>? Packages { get; set; }
 	}
 
 	public class Package
 	{
-		[JsonProperty ("javaId")]
+		[JsonPropertyName ("javaId")]
```

Comment:
@jonathanpeppers are these needed if we have a JsonSerializerContext? Its not clear in the docs

Reply:
I think it would put `"JavaId"` in the JSON without the attribute.

There might be a global setting though, for camelCase vs ProperCase.